tests/integ/test_tfs.py

```diff
@@ -36,28 +39,85 @@ def instance_type(request):
 def tfs_predictor(instance_type, sagemaker_session, tf_full_version):
     endpoint_name = sagemaker.utils.unique_name_from_base('sagemaker-tensorflow-serving')
     model_data = sagemaker_session.upload_data(
-        path='tests/data/tensorflow-serving-test-model.tar.gz',
+        path=os.path.join(tests.integ.DATA_DIR, 'tensorflow-serving-test-model.tar.gz'),
         key_prefix='tensorflow-serving/models')
-    with tests.integ.timeout.timeout_and_delete_endpoint_by_name(endpoint_name, sagemaker_session):
+    with tests.integ.timeout.timeout_and_delete_endpoint_by_name(endpoint_name,
+                                                                 sagemaker_session):
         model = Model(model_data=model_data, role='SageMakerRole',
                       framework_version=tf_full_version,
                       sagemaker_session=sagemaker_session)
         predictor = model.deploy(1, instance_type, endpoint_name=endpoint_name)
         yield predictor
 
 
+def tar_dir(directory):
+
+    tmp = tempfile.mkdtemp()
+
+    source_files = [os.path.join(directory, name) for name in os.listdir(directory)]
```

Comment:
shouldn't need to do this. method in utils should just take a dir path and do the right thing

Reply:
`sagemaker.utils.create_tar_file` receives an array of files, not necessarily under the same tree structure. Are you suggesting to create a new function that takes a folder instead?

Reply:
i think the create_tar_file should be changed... still not convinced we need to be generating these file lists this way (see arcname comment)